scripts/cmake/ThirdPartyLibVersions.cmake

```diff
@@ -1,7 +1,7 @@
 set(OGS_BOOST_URL "http://sourceforge.net/projects/boost/files/boost/1.58.0/boost_1_58_0.tar.bz2/download")
 set(OGS_BOOST_MD5 "b8839650e61e9c1c0a89f371dd475546")
 
-set(OGS_EIGEN_URL "http://bitbucket.org/eigen/eigen/get/3.2.5.tar.gz")
+set(OGS_EIGEN_URL "http://opengeosys.s3.amazonaws.com/ogs6-lib-sources/eigen-3.2.5.tar.gz")
```

Comment:
CMake internally uses curl to download which per default does not follow redirections. The BitBucket http-url gets redirected to http**s** and the CMake internal curl does not support https.

I can share S3 credentials if you want.